upup/pkg/fi/cloudup/awstasks/iaminstanceprofilerole.go

```diff
@@ -126,12 +126,14 @@ func (_ *IAMInstanceProfileRole) RenderTerraform(t *terraform.TerraformTarget, a
 }
 
 type cloudformationIAMInstanceProfile struct {
+	InstanceProfileName *string `json:"InstanceProfileName"`
 	//Path  *string              `json:"name"`
 	Roles []*cloudformation.Literal `json:"Roles"`
 }
 
 func (_ *IAMInstanceProfileRole) RenderCloudformation(t *cloudformation.CloudformationTarget, a, e, changes *IAMInstanceProfileRole) error {
 	cf := &cloudformationIAMInstanceProfile{
+		InstanceProfileName: e.Name,
```

Comment:
Terraform uses `e.InstanceProfile.Name` (Line 121) perhaps we should be consistent?

Reply:
Good point. Updated.